src/emulator/storage/apis/gcloud.ts

```diff
@@ -411,3 +412,8 @@ function sendObjectNotFound(req: Request, res: Response): void {
     });
   }
 }
+
+function getValidName(query: Query, metadata: IncomingMetadata): string | undefined {
```

Comment:
tiniest of nits, would also prefer a more descriptive name, go/java-practices/naming#method-name-expectations

What is a valid name? In what contexts should you use this? Something like `getIncomingFileNameFromRequest` is a little easier to parse esp if we're not adding a comment

Reply:
Makes sense, thanks for the suggestion too! I'll add it in